hscontrol/types/users.go

```diff
@@ -16,19 +18,57 @@ import (
 // that contain our machines.
 type User struct {
 	gorm.Model
+
+	// Username for the user, is used if email is empty
+	// Should not be used, please use Username().
 	Name string `gorm:"unique"`
```

Comment:
Same here, `Name`  is the user full name (given + family)... not necesarily unique.

https://openid.net/specs/openid-connect-basic-1_0-22.html#id_res

Reply:
This is the username, or, in OIDC terms: `preferred_username`, I suppose there is a risk that the upstream does not enforce uniqueness here, but also, we are kind of at the mercy of the non-oidc setup where we have had this unique all the time.

The Name you are referred to is going under DisplayName.

Reply:
https://github.com/juanfont/headscale/pull/2020/files#diff-7c0d50dc95e3b04d35f8bfbb30d4aee47101503fa703ea61747cb1a97c778d28R128 

was actually pointing at the wrong field, so fixed that.